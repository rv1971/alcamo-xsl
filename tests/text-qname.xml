<?xml version="1.0" encoding="UTF-8"?>

<?xml-stylesheet href="#xslt" type="text/xml"?>

<a:doc xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <xsl:stylesheet
      xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
      xml:id="xslt"
      version="1.0">
    <xsl:import href="../xsl/text.xsl"/>

    <xsl:template match="/">
      <a:result>
        <xsl:apply-templates select="/a:doc/a:data"/>
        <xsl:apply-templates select="/a:doc/a:data/@*"/>
     </a:result>
    </xsl:template>

    <xsl:template match="a:item|a:data/@*">
      <a:item>
        <xsl:value-of select="."/>
        <xsl:value-of select="' '"/>
        <xsl:apply-templates select="." mode="a:local-name"/>
        <xsl:value-of select="' '"/>
        <xsl:apply-templates select="." mode="a:ns-name"/>
      </a:item>
    </xsl:template>
  </xsl:stylesheet>

  <a:data
      xmlns="http://www.example.biz"
      xmlns:bar="http://bar.example.org"
      x1="foo"
      x2="bar:baz"
      x3=":qux">
    <a:item>foo</a:item>
    <a:item>bar:baz</a:item>
    <a:item>:qux</a:item>
  </a:data>
</a:doc>
