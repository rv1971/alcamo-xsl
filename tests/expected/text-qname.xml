<?xml version="1.0"?>
<a:result xmlns:a="tag:rv1971@web.de,2021:alcamo-xsl#">
  <a:item>foo foo</a:item>
  <a:item>bar:baz baz</a:item>
  <a:item>:qux qux</a:item>
</a:result>
